crates/noirc_frontend/src/hir/resolution/resolver.rs

```diff
@@ -722,9 +723,14 @@ impl<'a> Resolver<'a> {
         }
     }
 
-    pub fn resolve_global_let(&mut self, let_stmt: crate::LetStatement) -> HirStatement {
+    pub fn resolve_global_let(
+        &mut self,
+        let_stmt: crate::LetStatement,
```

Comment:
why `crate::LetStatement` instead of importing and doing `LetStatement`?

Reply:
I'm not sure, the code was this way before and I didn't change it. The addition of a new parameter made the formatter split it on multiple lines though.